Module not found: Error: Can't resolve 'buffer'

Hi, I believe https://github.com/diegomura/react-pdf/pull/1891 should fix problem with create-react-app v5 using webpack 5. But unfortunately when creating fresh cra application with instruction from quick-start https://react-pdf.org/ I see following errors: 

```
Failed to compile.

Module not found: Error: Can't resolve 'buffer' in '/Users/user/Documents/Code/cra-pdf-test/node_modules/base64-to-uint8array'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "buffer": require.resolve("buffer/") }'
	- install 'buffer'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "buffer": false }
ERROR in ./node_modules/base64-to-uint8array/index.js 8:10-32
Module not found: Error: Can't resolve 'buffer' in '/Users/user/Documents/Code/cra-pdf-test/node_modules/base64-to-uint8array'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "buffer": require.resolve("buffer/") }'
	- install 'buffer'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "buffer": false }

webpack compiled with 1 error
Files successfully emitted, waiting for typecheck results...
Issues checking in progress...
No issues found.
```

here is reproduction repo: https://github.com/Gratify-Ocelot/cra-pdf-test



_Originally posted by @Gratify-Ocelot in https://github.com/diegomura/react-pdf/issues/1891#issuecomment-1240563473_


here is proposed fix by @carlobeltrame https://github.com/diegomura/react-pdf/pull/1891#issuecomment-1240631229

--- a/.changeset/lemon-cameras-visit.md
+++ b/.changeset/lemon-cameras-visit.md
@@ -1,0 +1,5 @@
+---
+'@react-pdf/font': patch
+---
+
+Fix warning about 'buffer' dependency in CRA 5 and webpack 5

--- a/packages/font/package.json
+++ b/packages/font/package.json
@@ -24,7 +24,6 @@
   "dependencies": {
     "@babel/runtime": "^7.16.4",
     "@react-pdf/types": "^2.0.9",
-    "base64-to-uint8array": "^1.0.0",
     "cross-fetch": "^3.1.5",
     "fontkit": "^2.0.2",
     "is-url": "^1.2.4"

--- a/packages/font/src/font.js
+++ b/packages/font/src/font.js
@@ -3,7 +3,6 @@
 import isUrl from 'is-url';
 import fetch from 'cross-fetch';
 import * as fontkit from 'fontkit';
-import toUint8Array from 'base64-to-uint8array';
 
 const FONT_WEIGHTS = {
   thin: 100,
@@ -60,7 +59,12 @@ class FontSource {
 
     if (isDataUrl(this.src)) {
       const raw = this.src.split(',')[1];
-      this.data = fontkit.create(toUint8Array(raw), postscriptName);
+      const uint8Array = new Uint8Array(
+        atob(raw)
+          .split('')
+          .map(c => c.charCodeAt(0)),
+      );
+      this.data = fontkit.create(uint8Array, postscriptName);
     } else if (BROWSER || isUrl(this.src)) {
       const { headers, body, method = 'GET' } = this.options;
       const data = await fetchFont(this.src, { method, body, headers });

